internal/uvm/scsi.go

```diff
@@ -67,6 +73,10 @@ type SCSIMount struct {
 	// read-only layers. As RO layers are shared, we perform ref-counting.
 	isLayer  bool
 	refCount uint32
+	// specifies if this is a readonly layer
+	ReadOnly bool
+	// "VirtualDisk" or "PassThru" disk attachment type.
+	AttachmentType string
```

Comment:
Do these need to be exported?

Reply:
Same as that of the above comment on VSMBShare.